content/en/docs/concepts/workloads/pods/_index.md

```diff
@@ -257,8 +257,9 @@ section.
 
 ## Privileged mode for containers
 
-Any container in a Pod can enable privileged mode, using the `privileged` flag on the [security context](/docs/tasks/configure-pod-container/security-context/) of the container spec. This is useful for containers that want to use operating system administrative capabilities such as manipulating the network stack or accessing hardware devices.
-Processes within a privileged container get almost the same privileges that are available to processes outside a container.
+Any container in a Pod can enable privileged mode, using either the `privileged` (Linux) or `hostProcess` (Windows) flag on the [security context](/docs/tasks/configure-pod-container/security-context/) of the container spec. This is useful for containers that want to use operating system administrative capabilities such as manipulating the network stack or accessing hardware devices.
```

Comment:
AIUI this is only valid for clusters with `WindowsHostProcessContainers` enabled.

Reply:
I think we will need to have a separate wording here for Windows and Linux.  At this time, For Windows _every_ container must be hostprocess not any container.  The security context is slight different for windows as well (under the windowsOptions)

```
  securityContext:
    windowsOptions:
      hostProcess: true
```

Reply:
Yes, the feature is alpha. Should we update to or `hostPorcess` (Windows) flag _if WindowsHostProcessContainers feature is enabled for the cluster_?